.travis.yml

```diff
@@ -46,15 +46,15 @@ install:
   - git submodule update
   - if [ $CXX == g++ ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then CXX=g++-7; fi
   - if [ $CXX == clang++ ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then CXX=clang++-5.0; fi
-  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew install libelf zlib; fi
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew install libelf zlib gcc@7; fi
```

Comment:
There is no build with GCC on OS X, we use only Clang/LLVM for that environment.